packages/magento-customer/components/AccountSignInUpForm/AccountSignInUpForm.tsx

```diff
@@ -1,27 +1,15 @@
-import {
-  Button,
-  FormDiv,
-  FormActions,
-  FormRow,
-  LayoutTitle,
-  extendableComponent,
-} from '@graphcommerce/next-ui'
-import { emailPattern } from '@graphcommerce/react-hook-form'
-import { Trans } from '@lingui/react'
-// eslint-disable-next-line @typescript-eslint/no-restricted-imports
-import { Box, CircularProgress, Link, SxProps, TextField, Theme, Typography } from '@mui/material'
-import { useRouter } from 'next/router'
-import { useEffect } from 'react'
-import { CustomerDocument, useFormIsEmailAvailable } from '../../hooks'
+import { SxProps, Theme } from '@mui/material'
+import { CustomerDocument } from '../../hooks'
 import { useCustomerQuery } from '../../hooks/useCustomerQuery'
-import { ApolloCustomerErrorAlert } from '../ApolloCustomerError'
-import { SignInForm } from '../SignInForm/SignInForm'
-import { SignUpForm } from '../SignUpForm/SignUpForm'
+import { AccountSignInUpIsEmailAvailableForm } from './AccountSignInUpIsEmailAvailableForm'
+import { AccountSignInUpToggleForm } from './AccountSignInUpToggleForm'
 
-export type AccountSignInUpFormProps = { sx?: SxProps<Theme> }
+// When the guestCheckout is disabled in Magento add loginMethod "TOGGLE" to your config to enable the toggle login / register flow which is mandatory.
+// Toggle flow also disables the isEmailAvailable call to the magento backend.
+
+// Default is set to IS_EMAIL_AVAILABLE
```

Comment:
Isn't the guestCheckout value retrievable via the store config in Magento? If so, we should toggle this behaviour automatically, since wrong configuration could lead to faulty UI

Reply:
No the guestCheckout value is not retrievable via the store config in Magento. If we set guestCheckout disabled in the config the IsEmailAvailable API will always return true.